src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.resx

```diff
@@ -123,7 +123,7 @@
   <data name="ConfigCommandExamples" xml:space="preserve">
     <value>nuget config -Set HTTP_PROXY=http://127.0.0.1 -Set HTTP_PROXY.USER=domain\user
 nuget config HTTP_PROXY</value>
-    <comment>Please don't localize this string</comment>
```

Comment:
How do you consider as `three letter suffixed` string? Exact where is 3 letter suffix?

Reply:
WE get the three-letter-suffix from this API : https://docs.microsoft.com/en-us/dotnet/api/system.globalization.cultureinfo.threeletterwindowslanguagename?view=net-6.0

https://github.com/NuGet/NuGet.Client/blob/421801a875b876882ffe24d87e7ea0862bb1b41f/src/NuGet.Clients/NuGet.CommandLine/Common/LocalizedResourceManager.cs#L23-L36